dodo.1

```diff
@@ -11,7 +11,7 @@ dodo - a scriptable in-place file ditor
 
 .SH DESCRIPTION
 dodo is a non-interactive scriptable in place file editor.
-It takes exactly 1 argument, the file to work on, and it takes its program input from stdin.
+It takes the file to work on as an argument, and it takes its program input from stdin.
 .P
 dodo was born from the need to efficiently edit very large files (16GB plain sql dumps),
 I was unable to so using my normal toolset (ed, sed or vim) and I realised that writing a tool
```

Comment:
found a typo here that I wrote a long time ago, should be "I was unable to DO so using my normal toolset"